.gitignore

```diff
@@ -1,6 +1,7 @@
 # Files and directories created by pub
 .dart_tool/
 .packages
+pubspec.lock
```

Comment:
Why do you want to remove this? I prefer committing the lock file as that's the whole purpose of having one.

Reply:
Hm, I've read the docs and you're right:

```
For library packages, don’t commit the pubspec.lock file. Regenerating the pubspec.lock file lets you test your package against the latest compatible versions of its dependencies.
```

Even though I don't really agree with this because one of my dependencies can one day simply break my build by making a mistake in their sem-versioning, I will accept the change to be in line with the standard recommendation :/